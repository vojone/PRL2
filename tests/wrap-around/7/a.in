0000000000
0011101000
0011101000
0001101000
0000111000
0000000000
0000000000
0000000000
0000000000
0000000000
