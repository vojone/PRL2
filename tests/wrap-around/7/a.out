0000000000
0000000000
0000000000
0000000000
0000000000
0000000000
0000000000
0001000000
0001000000
0001000000
